components/Profile/ProfileStatsBox.tsx

```diff
@@ -57,9 +50,10 @@ const ProfileStatsBox = ({
               <Image source={require("../../assets/images/profileCoin.png")} />
 
               <Text
-                className="font-sans-bold mt-2 text-projectWhite"
+                className="text-caption-lg-bold mt-2"
```

Comment:
This doesn't seem to be a preset in the typography Figma. Possibly we removed this preset when we removed your changes in `theme/typography.ts`